### PR TITLE
Tlp updates

### PR DESCRIPTION
**Pull request by objectadjective**
This should account for the changes on our list (aside from video on homepage which was previously requested as a modal overlay).

### CODE REVIEW

File: _src/_includes/themes/apache/home.html [hunk 1]
```diff
@@ -71,14 +70,13 @@
       <img src="/assets/img/robotTutorialSm.png" alt="What is Apache SystemML?">
     </div>
     <div class="col col-6 content-group content-group--more-padding button-group">
-      <h2>Get Started</h2>
-      <p>New to Apache SystemML? Try our Get Started tutorial that will walk you through setting up your environment and getting you up and going with SystemML.</p>
+      <h2>Install SystemML</h2>
+      <p>New to Apache SystemML? Try our guick install guide that will walk you through setting up your environment and getting you up and going with SystemML.</p>
```
Comment (dusenberrymw) [inline _src/_includes/themes/apache/home.html]: typo: "guick" -> "quick"

File: _src/_includes/themes/apache/home.html [hunk 2]
```diff
@@ -71,14 +70,13 @@
...
       <a class="button button-primary" href="install-systemml.html">Install SystemML</a>
       <a class="button button-secondary" href="documentation.html" target="_blank">Docs</a>
     </div>
   </div>
-
-  <h4 class="text-center"><a href="get-started.html">View Sample Notebooks</a></h4>
+  <h4 class="text-center"><a href="get-started.html#sample-notebook">Veiew Sample Notebooks</a></h4>
```
Comment (dusenberrymw) [inline _src/_includes/themes/apache/home.html]: typo "Veiew" -> "View"

File: _src/assets/img/jupyter-logo.svg [hunk 3]
```diff
@@ -1 +1,5 @@
...
+>>>>>>> 2137dcfd320076c41ce610930ae5a1e5d48d5cb8
```
Comment (dusenberrymw) [inline _src/assets/img/jupyter-logo.svg]: @gweidner Had a good find here.  This svg file has conflict text in it, which is why it wasn't working.  Can you just replace this svg with the correct svg?  I think it's either line 2 or 4 (or just replace the file), and then the svg should work again.

**Comment by deroneriksson** [comment]
LGTM. I'll merge and make the @dusenberrymw and @gweidner updates when I merge. Thank you @objectadjective!

**Pull request closed without merging**